modules/rostests/apitests/ntdll/RtlRemoteCall.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Test for x64 and x86 RtlRemoteCall
+ * COPYRIGHT:   Copyright 2025 Ratin Gao <ratin@knsoft.org>
+ */
+
+#include "precomp.h"
+
+EXTERN_C void _stdcall RtlRemoteCall_TestProc(void);
+
+static
+ULONG
+NTAPI
+ThreadStartStub(
+    _In_ PVOID ThreadParameter)
+{
+    UNREFERENCED_PARAMETER(ThreadParameter);
+    return STATUS_SUCCESS;
+}
+
+static
+NTSTATUS
+RemoteCall(
+    _In_ PVOID CallSite,
+    _In_ ULONG ArgumentCount,
+    _In_reads_(ArgumentCount) PULONG_PTR Arguments,
+    _In_ BOOLEAN PassContext)
+{
+    NTSTATUS Status;
+    HANDLE ThreadHandle;
+    CONTEXT Context;
+    THREAD_BASIC_INFORMATION tbi;
+
+    /* Create new thread in suspended state */
+    Status = RtlCreateUserThread(NtCurrentProcess(),
```

Comment:
For a function with `Remote` in the name, I would have expected at least one test on another process?